package/yast2-multipath.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Jan 26 12:26:32 UTC 2015 - jreidinger@suse.com
+
+- fix removing \r character accidentally removed during conversion
```

Comment:
`removing...removed character` sounds weird to me, I'd rather write something like `fixed \r character lost during conversion to Ruby`